### PR TITLE
FT: Update README for local usage

### PR DESCRIPTION
**Pull request by bennettbuchanan**
Some README changes needed when starting from new repositories, and using replication locally.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,12 +123,13 @@ chmod 400 ./tests/utils/keyfile && \
...
 ```
 
-In a new shell, start the Scality S3 server:
+In a new shell, start the Scality S3 server with the record log enabled:
 
 ```
 git clone https://github.com/scality/s3 ~/replication/s3 && \
 cd ~/replication/s3 && \
 npm i && \
+sed -i '' 's/"enabled": false.*/"enabled": true,/' config.json \
```
Comment (jonathan-gramain) [inline README.md]: This is too hacky IMO, I would advise to split this in three steps, with explaining what to change in the config manually in between and an explanation of what this change does to inform the user.

`jq` would be a nice alternative though it messes up the sections and is not part of the bare unix command line tools so has to be installed first, but why not. Also `sed` has a way to define boundaries as regexp where to do the substitutions which would be safer but still hacky.

Comment (bennettbuchanan) [reply README.md]: Agree. I will update this to just explain the change manually.

File: README.md [hunk 2]
```diff
@@ -186,10 +187,20 @@ Default region name [None]:
...
+~/replication/backbeat/bin/localReplication.js \
+```
+
 Set up replication on your buckets:
 
 ```
-node ~/replication/backbeat/bin/setupReplication.js setup \
+node ~/replication/backbeat/bin/localReplication.js setup \
 --source-bucket source-bucket \
 --source-profile backbeatuser \
```
Comment (bennettbuchanan) [inline README.md]: 'backbeatuser' should be changed to add profiles 'bart' for source and 'lisa' for destination

**Review by dora-korpar — COMMENTED** [review]
~~Could you also update references to the kafka version? It looks like `kafka_2.11-0.11.0.0` is no longer available. It's `kafka_2.11-0.11.0.2` now.
References are on lines 63, 65, 66, 72, and 78~~

Done!

**Review by bennettbuchanan — COMMENTED** [review]
We need to also update the `npm run queue_processor` command to take a site argument, and also run the `npm run replication_status_processor` in parallel.

**Comment by bennettbuchanan** [comment]
The README would require removal of these changes now in any case. Closing this.

**Pull request closed without merging**